### PR TITLE
Sean connect backend frontend

### LINKED ISSUE #15: Integrate React-Native with Django Backend
**Describe the solution you'd like**
Send a GET request from React-Native code and receive a response from Django, whether that's a hard coded JSON object or the actual database.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by seansisson**
- Added django-cors-headers to Django backend to allow all cross origin requests (for now)
- Simplify react-native frontend
- Make API call from frontend to get posts data
- Display data on page if retrieved successfully

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -210,14 +210,14 @@ RNTester/Pods/*
...
-*.jks
-*.p8
-*.p12
-*.key
-*.mobileprovision
-*.orig.*
-web-build/
-web-report/
+/react-native/node_modules/**/*
+/react-native/.expo/
```
Comment (SimonBartonPSU) [inline .gitignore:214]: Ah of course! @EHaake this solves the gitignore issue. 

Good thinking @seansisson

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+server:
+	docker-compose build
+	docker-compose up
+
+js:
+	cd react-native && expo start
+
+console:
+	docker exec -it djapi bash
+
```
Comment (SimonBartonPSU) [inline Makefile:12]: Very nice, Makefile is going to help in the long run for sure. Better to start now and build as we go.

three commands `make server` which runs the docker compose commands,
 `make js` which starts expo
 and `make console`. 

Default `make` is equivalent to `make server`, beautiful

If we wanted to get real fancy, in another task we could have the `make js` run a command and parse output to determine local private IP then plop that into App.js for us automatically

File: django/todoproj/settings.py [hunk 3]
```diff
@@ -51,6 +53,8 @@
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 ]
 
+CORS_ORIGIN_ALLOW_ALL = True
```
Comment (SimonBartonPSU) [inline django/todoproj/settings.py:56]: CORS makes things easier for front end generally?

Or GCP related? Or both?

File: react-native/App.js [hunk 4]
```diff
@@ -1,86 +1,63 @@
...
-     TextInput,
-     Button,
-} from "react-native";
+import React from "react";
+import {StyleSheet, Text, View, ActivityIndicator} from "react-native";
 
-export default class App extends Component {
-  constructor(props) {
-  super(props);
+export default class App extends React.Component {
```
Comment (SimonBartonPSU) [inline react-native/App.js:6]: Nice

**Review by SimonBartonPSU — COMMENTED** [review]
Looking really great, nice work. gitignore changes are good. big derp on my part there. Make commands are running for me.

But with containers running and Expo running, I'm stuck with the App just loading on both my iPhone and the 'W' web viewer. Not sure if this is a me problem or a code problem, but was result of initial attempt

loading

**Comment by seansisson** [comment]
This branch can be safely nuked from low earth orbit, is resolved by #39

**Comment by EHaake** [comment]
Closed without merging because changes were added to separate pull request

**Pull request closed without merging**